access_dynamo_from_python.py

```diff
@@ -0,0 +1,30 @@
+import json
+import boto3
+from boto3.dynamodb.conditions import Key
+
+def lambda_handler(event, context):
+        # TODO implementimport boto3
+    
+    TABLE_NAME = "Personas"
+    
+    # Creating the DynamoDB Client
+    dynamodb_client = boto3.client('dynamodb', region_name="sa-east-1")
+    
+    # Creating the DynamoDB Table Resource
+    dynamodb = boto3.resource('dynamodb', region_name="sa-east-1")
+    table = dynamodb.Table(TABLE_NAME)
+    response = table.get_item(
+        Key={
+            'Id': '0',
+            'Edad': '22'
```

Comment:
might be a good idea to receive these as params to your lambda function (in the event)